src/tasks/actions/thunks.ts

```diff
@@ -592,3 +617,15 @@ export const runDuration = (finishedAt: Date, startedAt: Date): string => {
 
   return diff + ' ' + timeTag
 }
+
+export const setTasksPageAsCurrent = () => async (
+  dispatch: Dispatch<Action>
+) => {
+  dispatch(setCurrentTasksPage('TasksPage'))
+}
+
+export const setTaskRunsPageAsCurrent = () => async (
+  dispatch: Dispatch<Action>
+) => {
+  dispatch(setCurrentTasksPage('TaskRunsPage'))
+}
```

Comment:
let's remove these functions as they're not necessary.

Thunks exist to handle [side effects](https://en.hexlet.io/courses/intro_to_programming/lessons/pure/theory_unit) when dispatching redux actions. Other thunks in our codebase have side effects like making a request, or updating a file. Since these only change the value of the redux state, these are pure actions and don't need a thunk. Dispatching an action is a UI level concern that components should handle.

In the places you call the thunk, instead just call the actions and pass in the proper value.

Reply:
that makes sense. I've removed them and called the action in TaskRunsCard and TaskCard and passed the respective values